src/blockchain/ain/jellyfish/jellyfish.service.ts

```diff
@@ -102,20 +103,46 @@ export class JellyfishService {
   }
 
   async rawTxForSendFromLiq(to: string, amount: BigNumber): Promise<RawTxDto> {
-    // TODO (Krysh) implement raw tx
-    return {
-      hex: '',
-      scriptHex: '',
-      prevouts: [],
-    };
+    return this.rawTxForSend(Config.staking.liquidity.address, to, amount, true);
   }
 
   async rawTxForSendToLiq(from: string, amount: BigNumber): Promise<RawTxDto> {
-    // TODO (Krysh) implement raw tx
+    return this.rawTxForSend(from, Config.staking.liquidity.address, amount, false);
+  }
+
+  private async rawTxForSend(from: string, to: string, amount: BigNumber, addChange: boolean): Promise<RawTxDto> {
```

Comment:
We need to keep in mind, that not the full amount will arrive, if `addChange = false`.

Reply:
maybe we should improve the naming. For me it implies that without `change` there are no UTXOs returned

Reply:
just inverted the functionality and renamed it to `sendFullAmount`